### ISSUE TITLE
Apple M1 arm64 Varnish not starting not starting

### ISSUE THREAD

**Issue by jeroenbegyn** [docksal/service-varnish#45]
I am getting this error:

```
/usr/local/bin/docker-entrypoint.sh: running /etc/docker-entrypoint.d/05-config-templates.sh
Generating config from a custom VCL in /var/www/.docksal/etc/varnish/default.vcl...
/etc/docker-entrypoint.d/05-config-templates.sh: line 17:     8 Illegal instruction     gomplate --file ${vcl_template} --out /etc/varnish/default.vcl
```

It seems to be related to this issue: https://github.com/docksal/service-vhost-proxy/issues/88

#46 This PR fixes the issue for me.

**Comment by jeroenbegyn** [comment]
#46 This PR fixes the issue for me.

**Comment by lmakarov** [comment]
Addressed in https://github.com/docksal/service-varnish/pull/46

**Comment by lmakarov** [comment]
Released as a hotfix in [v3.1.1](https://github.com/docksal/service-varnish/releases/tag/v3.1.1).

@jeroenbegyn Can you please confirm Varnish works for you now on M1/ARM64?
